internal/common/baseclient/iito.go

```diff
@@ -47,13 +54,13 @@ func (c *BaseClient) UpdateGiftInfo(acceptedGifts map[shared.ClientID]shared.Gif
 // SendGift is executed at the end of each turn and allows clients to
 // send the gifts promised in the IITO
 // COMPULSORY, you need to implement this method
-func (c *BaseClient) SendGift(receivingClient shared.ClientID, amount int) error {
+func (c *BaseClient) SendGift(offer shared.GiftOffer) error {
 	return nil
 }
```

Comment:
Is this meant to be nil?
- If it is meant to be nil, can you include this in the comments
- If not - Is it possible to add a TODO?

Same for below

Reply:
This whole file contains placeholder implementations. The appropriate functions are marked COMPULSORY, which means the implementation we put in here are "useless". In the previous functions we put something down so that agent implementers can read it and have a good idea of what the function is supposed to do, you'll notice that the implementations thus far just had (mostly) the correct looping and data structure logic; they're not supposed to do anything "meaningful" from a gameplay standpoint.

Therefore in this case the only thing we really can do is just return no error and do nothing with the gift offer we received. Open to change of course.